src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java

```diff
@@ -3,14 +3,19 @@
 import org.broadinstitute.hellbender.cmdline.Argument;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
+import java.io.Serializable;
+
 /**
  * Keep reads that are within a given max fragment length.
  */
-public final class FragmentLengthReadFilter implements ReadFilter {
+public final class FragmentLengthReadFilter extends ReadFilter implements Serializable  {
     private static final long serialVersionUID = 1l;
 
-    @Argument(fullName = "maxFragmentLength", shortName = "maxFragment", doc="Keep only read pairs with fragment length at most equal to the given value", optional=true)
-    public int maxFragmentLength = 1000000;
+    @Argument(fullName = "maxFragmentLength",
+            shortName = "maxFragment",
+            doc = "Keep only read pairs with fragment length at most equal to the given value",
+            optional = false)
+    public Integer maxFragmentLength; // no default value, since the command line parser will think its set
```

Comment:
Yes, all the individual filters have been updated to have defaults where appropriate.